helpdesk_mgmt/__manifest__.py

```diff
@@ -14,6 +14,7 @@
     "Domatix, "
     "Factor Libre, "
     "SDi Soluciones, "
+    "Mind And Go, "
```

Comment:
I'm afraid these improvements doesn't qualify for co-authorship, just for contributors.

Reply:
thanks @pedrobaeza 
Done !